src/dtgtk/thumbtable.c

```diff
@@ -2043,6 +2045,7 @@ void dt_thumbtable_update_accels_connection(dt_thumbtable_t *table, const int vi
 static gboolean _filemanager_ensure_rowid_visibility(dt_thumbtable_t *table, const int rowid)
 {
   if(rowid < 1) return FALSE;
+  if(!table->list || g_list_length(table->list) == 0) return FALSE;
```

Comment:
The first tests is superfluous as g_list_length (as most -all?- g_ routines are coping with NULL). But I'll merge as-is, that was just a remark.